### PR TITLE
Android degoogling

### PR DESCRIPTION
**Pull request by milleniumbug**

### CODE REVIEW

File: Android.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- Bootloader unlock
+- Installing alternative ROMs
+
+# LineageOS
+
+This is what CyanogenMod is called now, it's a fork. It's also most likely candidate as an alternative to your stock ROM. As a side effect of legal fights with Google it doesn't include any of the closed source Google applications, which is a good thing for us. It has the highest chance of working with your phone.
+
+# Replicant
+
+Open source-only distro of Android, which means it has even less chance of working on your device.
```
Comment (thecoshman) [inline Android.md]: 'even less' is comparative, what are you comparing it to?

These are two possible alternate ROMs so perhaps put them under a common `#` heading and put these at a lower sub heading level

**Review by thecoshman — CHANGES_REQUESTED** [review]

File: Android.md (hunk 1 above)
Comment (nabijaczleweli) [reply Android.md]: >what are you comparing it to?

LineageOS, probably.

Comment (milleniumbug) [reply Android.md]: Yeah, the intent was to compare it with LineageOS, but I'll amend this.

File: Android.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Replicant
+
+Open source-only distro of Android, which means it has even less chance of working on your device than LineageOS.
+
+# App repositiories
+
+Since Google Play relies on Google Play Services a.k.a. all the Google stuff we want to avoid, we won't be able to use it, or at least, not directly through the Google Play apk. You may want to use alternative app repositories:
+
+- F-Droid. Hosts open source applications only, and is transparent about unwanted features in apps. 
+- Aptoide. Likely to have a ton of **unofficially** republished applications from Google Play. Trust them on your own risk.
```
Comment (nabijaczleweli) [inline Android.md]: "At your own risk", innit?

**Review by thecoshman — APPROVED** [review]

**Pull request merged**